src/lang/english.txt

```diff
@@ -1855,32 +1855,32 @@ STR_LIVERY_PRIMARY_TOOLTIP                                      :{BLACK}Choose t
 STR_LIVERY_SECONDARY_TOOLTIP                                    :{BLACK}Choose the secondary colour for the selected scheme. Ctrl+Click will set this colour for every scheme
 STR_LIVERY_PANEL_TOOLTIP                                        :{BLACK}Select a colour scheme to change, or multiple schemes with Ctrl+Click. Click on the box to toggle use of the scheme
 
-STR_LIVERY_DEFAULT                                              :Standard Livery
-STR_LIVERY_STEAM                                                :Steam Engine
-STR_LIVERY_DIESEL                                               :Diesel Engine
-STR_LIVERY_ELECTRIC                                             :Electric Engine
-STR_LIVERY_MONORAIL                                             :Monorail Engine
-STR_LIVERY_MAGLEV                                               :Maglev Engine
+STR_LIVERY_DEFAULT                                              :Standard livery
+STR_LIVERY_STEAM                                                :Steam engine
+STR_LIVERY_DIESEL                                               :Diesel engine
+STR_LIVERY_ELECTRIC                                             :Electric engine
+STR_LIVERY_MONORAIL                                             :Monorail engine
+STR_LIVERY_MAGLEV                                               :Maglev engine
 STR_LIVERY_DMU                                                  :DMU
 STR_LIVERY_EMU                                                  :EMU
-STR_LIVERY_PASSENGER_WAGON_STEAM                                :Passenger Coach (Steam)
-STR_LIVERY_PASSENGER_WAGON_DIESEL                               :Passenger Coach (Diesel)
-STR_LIVERY_PASSENGER_WAGON_ELECTRIC                             :Passenger Coach (Electric)
-STR_LIVERY_PASSENGER_WAGON_MONORAIL                             :Passenger Coach (Monorail)
-STR_LIVERY_PASSENGER_WAGON_MAGLEV                               :Passenger Coach (Maglev)
-STR_LIVERY_FREIGHT_WAGON                                        :Freight Wagon
+STR_LIVERY_PASSENGER_WAGON_STEAM                                :Passenger coach (Steam)
```

Comment:
I think "(Steam)" should probably also lose its capitalization here.

(ditto for following lines)